lib/src/main/java/grpcbridge/Bridge.java

```diff
@@ -75,8 +76,9 @@ public static BridgeBuilder builder() {
      *
      * @param routes list of available routes
      */
-    Bridge(List<Route> routes) {
+    Bridge(List<Route> routes, boolean preserveProtoFieldNames) {
```

Comment:
Do we want to generalize it and make it enum of different types of encoding?

Reply:
Or actually just accept the Printer here that you use below and have another constructor that uses a good default.

Reply:
Good question, yeah i think it makes sense to feed the `Printer` instance in